Makefile

```diff
@@ -89,9 +89,12 @@ INV_DIR := $(shell pwd)
 
 ifeq (cygwin,$(OSTYPE))
 #JAVA tools need Windows path on Windows
-JAR_DIR := $(shell cygpath -m $(INV_DIR))
+JAR_DIR :=`cygpath -wp $(INV_DIR)`
```

Comment:
As a minor issue, the space before and after the := lines is inconsistent.  Same two lines down.